src/Sentry/Laravel/EventHandler.php

```diff
@@ -534,6 +548,58 @@ private function cleanupScopeForOctane(): void
         $this->pushedOctaneScope = false;
     }
 
+    protected function httpClientResponseReceivedHandler(HttpClientEvents\ResponseReceived $event): void
+    {
+        if (!$this->recordHttpClientRequests) {
+            return;
+        }
+
+        $level = Breadcrumb::LEVEL_INFO;
+        if (!$event->response->successful()) {
```

Comment:
TBD what we should consider as an error.

Reply:
Seems like that should come from higher up and be defined in some Sentry document what is considered a successful response and not.

Reply:
Went with `LEVEL_WARNING` for 4XX and `LEVEL_ERROR` for 5XX